packages/subiquity_client/generator/Makefile

```diff
@@ -5,6 +5,8 @@ types_dart = ../lib/src/types.dart
 generate:
 	python3 generator.py $(types_py) --output $(types_dart)
 	dart format $(types_dart)
+	dart fix --apply $(types_dart)
+	dart format $(types_dart)
```

Comment:
Running `dart format` twice here is intended - the first is needed to trigger the 'missing trailing comma' lint, the second one reformats the code after the commas are added by `dart fix`